source/common/http/header_map_impl.cc

```diff
@@ -46,6 +46,13 @@ bool validatedLowerCaseString(absl::string_view str) {
   return lower_case_str == str;
 }
 
+absl::string_view delimiterByHeader(const LowerCaseString& key, bool correctly_coalesce_cookies) {
+  if (correctly_coalesce_cookies && key == Http::Headers::get().Cookie) {
+    return "; ";
+  }
+  return ",";
```

Comment:
perf nit: I don't know if the compiler is smart enough to make the string_view ahead of time, so this might require a strlen each time. I would define constant string_view and then return them.

Reply:
Get it.